templates/indieauth-authenticate-form.php

```diff
@@ -1,49 +1,37 @@
 <?php
-$errors = new WP_Error();
+$login_errors = new WP_Error();
 login_header(
-	sprintf( __( 'Authenticate %1$s', 'indieauth' ), empty( $client_name ) ? $client_id : $client_name ),
+	/* translators: Client Name or ID */
+	sprintf( __( 'Authenticate %1$s', 'indieauth' ), empty( $client_name ) ? esc_url( $client_id ) : $client_name ),
 	'',
-	$errors
+	$login_errors
 );
-$user_id = get_url_from_user( $current_user->ID );
-if ( ! $user_id ) {
+$user_website = esc_url( get_url_from_user( $current_user->ID ) );
+if ( ! $user_website ) {
 	__e( 'The application cannot sign you in as WordPress cannot determine the current user', 'indieauth' );
 	exit;
 }
-	
+
 ?>
-<form method="post" action="<?php echo $url; ?>">
+<form method="post" action="<?php echo esc_url( $url ); ?>">
 	<div class="user-info">
 		<?php echo get_avatar( $current_user->ID, '48' ); ?>
 		<?php
 			printf(
-				'<p>' . __( 'The app <strong>%1$s</strong> would like to identify you as <strong>%2$s</strong>, which is user %3$s(%4$s).', 'indieauth' ) . '</p>',
-				$client,
-				$user_id,
-				$current_user->display_name,
-				$current_user->user_nicename
-				
+				/* translators: 1. Client with link 2. User ID 3. User Display Name 4. User Nicename */
+				'<p>' . esc_html__( 'The app %1$s would like to identify you as %2$s, which is user %3$s(%4$s).', 'indieauth' ) . '</p>',
+				$client, // phpcs:ignore
+				'<strong>' . esc_url( $user_website ) . '</strong>',
+				'<strong>' . esc_html( $current_user->display_name ) . '</strong>',
+				esc_html( $current_user->user_nicename )
 			);
```

Comment:
This doesn't need to be phpcs:ignored. A possible escaped solution (mind that because of Github limitations this doesn't result in correct code, the prepending `<?php printf` needs to be replaced as well) would look like this:
```suggestion
		<p>
		<?php
			echo wp_kses(
				sprintf(
				/* translators: 1. Client with link 2. User ID 3. User Display Name 4. User Nicename */
					__( 'The app %1$s would like to identify you as %2$s, which is user %3$s (%4$s).', 'indieauth' ),
					$client,
					'<strong>' . esc_url( $user_website ) . '</strong>',
					'<strong>' . $current_user->display_name . '</strong>',
					$current_user->user_nicename
				),
				array( 'strong' => array() )
			);
			?>
		</p>
```